convert2rhel/checks.py

```diff
@@ -803,7 +813,7 @@ def is_loaded_kernel_latest():
                 "Could not find any %s from repositories to compare against the loaded kernel.\n"
                 "Please, check if you have any vendor repositories enabled to proceed with the conversion.\n"
                 "If you wish to ignore this message, set the environment variable "
-                "'CONVERT2RHEL_UNSUPPORTED_SKIP_KERNEL_CURRENCY_CHECK' to 1." % package_to_check
+                "'CONVERT2RHEL_ALLOW_SKIP_KERNEL_CURRENCY_CHECK' to 1." % package_to_check
             )
```

Comment:
One of the reasons to change away from using 'unsupported' is to also change the wording of the whole log message, not just the env var. You should lead the collaboration on how to word it here with the rest of the team but I can give some hints of why I don't think the message we have atm is the best

Overall it doesn't go into why this is an issue. Okay, we couldn't find any comparisons to see if it's the latest, but why does it matter? It also doesn't mention given what we tried to do, what happened to make this happen? (We know it's due to not returning any packages, but should be made evident in the logs too)

It goes into why it could have happened, being that the vendor repositories might not be enabled. But the message is a bit weird in that it tells you to do something to continue the conversion, but we already exited the conversion. We can't proceed from this point

Reply:
Cool thanks, I will get this PR set up as much as I can, then set up a GDoc to prepare for collaboration next year!